src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball(ColorSupplier.getRandomColor().toString(), new Random().nextInt(100));
```

Comment:
lets make ColorSupplier and Random a class-level variable - to avoid creating a new instance each time this method is called